pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -53,7 +53,7 @@ type ServiceBindingRequestStatus struct {
 	Conditions []conditionsv1.Condition `json:"conditions"`
 	// Secret is the name of the intermediate secret
 	Secret string `json:"secret"`
-	// ApplicationObjects contains all the application objects filtered by label
+	// ApplicationObjects contains all the application objects filtered by name or label
```

Comment:
How about simplifying even further: "List of bound applications" ? the declaration on line 58 states that.

Reply:
@pedjak "List of bound applications" is not very descriptive. It doesn't specify on what basis the applications were bound.